src/main/java/com/fasterxml/jackson/databind/MappingIterator.java

```diff
@@ -162,6 +162,14 @@ protected MappingIterator(JavaType type, JsonParser p, DeserializationContext ct
         }
     }
 
+    /**
+     * Copy-constructor that sub-classes can use when creating new instances
+     * by fluent-style construction
+     */
```

Comment:
I refer to the following comment
https://github.com/FasterXML/jackson-databind/blob/896c1cfd290e99526ed7c0627d3ca040d8884d44/src/main/java/com/fasterxml/jackson/databind/deser/std/StdValueInstantiator.java#L92-L95